src/config.js

```diff
@@ -63,7 +65,29 @@ function debug () {
   if (HEROKU_DEBUG) return parseInt(HEROKU_DEBUG)
   return 0
 }
+function skipAnalytics (userConfig: UserConfig) {
+  if (userConfig.skipAnalytics) {
+    return true
+  } else if (process.env['TESTING'] === '1') {
+    return true
+  }
+  return false
+}
+
+let loadUserConfig = function (configDir) {
+  try {
+    const config: UserConfig = fs.readJsonSync(path.join(configDir, 'user_config.json'), 'utf8')
+    return config
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      return { 'skipAnalytics': false }
+    } else {
+      throw e
+    }
+  }
+}
 
+let userConfig : UserConfig
```

Comment:
while for the CLI it's fine to make this a global variable, if we reused this library for chatops we would have to localize it. Better to define the variable inside `buildConfig` I think